docs/getting-started/zns-emulation.md

```diff
@@ -83,7 +99,7 @@ Node             SN                   Model                                    N
 /dev/nvme0n1     deadbeef             QEMU NVMe Ctrl                           1          34.36  GB /  34.36  GB      4 KiB +  0 B   1.0
 ```
 
-The *lsscsi* utility will also show the emulated NVMe device.
+The *lsscsi* utility shows the emulated NVMe device:
```

Comment:
I suggest removing the lsscsi example. The nvme command has provided the appropriate information.

Reply:
I've removed the lsscsi example.